lib/experimental/block-editor-settings-mobile.php

```diff
@@ -33,7 +67,64 @@ function gutenberg_get_block_editor_settings_mobile( $settings ) {
 		$settings['__experimentalEnableListBlockV2'] = true;
 	}
 
-	return $settings;
+	return keep_supported_block_editor_settings_mobile(
```

Comment:
I think `filter_by_supported_block_editor_settings_mobile` better communicates the intent. WDYT? Likely merely a personal preference, feel free to disregard.

Reply:
I like it, actually. Initially, I had "filter" in the name, and then second-guessed myself since the semantics of `addFilter` can mean more than selectively removing items, etc. (more akin to a middleware), but I was probably overthinking it 🤷‍♂️, and filter is just fine in the name. 👍